shardingsphere-infra/shardingsphere-infra-executor/src/main/java/org/apache/shardingsphere/infra/executor/sql/process/ExecuteProcessEngine.java

```diff
@@ -55,6 +55,13 @@ public static void initialize(final SQLStatementContext<?> context, final Execut
         }
     }
     
+    /**
+     * Clean up execution ID.
+     */
+    public static void cleanupExecutionID() {
+        ExecutorDataMap.getValue().remove(ExecuteProcessConstants.EXECUTE_ID.name());
```

Comment:
Could we move this  statement to `void finish(final String executionID) ` and remove this function?

Reply:
Could not. Since `void finish(final String executionID)` may be not executed on exception, but `void cleanupExecutionID()` must be executed after `void initialize(context, executionGroupContext)`.

Reply:
Maybe `initialize()`, `finish()`  and `clean()` could form as,
```java
  try {
          ExecuteProcessEngine.initialize(sqlStatementContext, executionGroupContext);
          List<Integer> results = jdbcLockEngine.execute(executionGroupContext, sqlStatementContext, routeUnits, callback); // Child report
          ExecuteProcessEngine.finish(); // Final report
          return result;
        } catch (final Exception ex) {
            ExecuteProcessEngine.clean();
        }
```